src/backend/bs_worker

```diff
@@ -2848,6 +2848,9 @@ sub dobuild {
 	    $kiwiorigins = {};			# but collect origins
 	  }
 	  $needsslcert = 1 if /^#\s*needssslcertforbuild\s*$/s;
+	  # On Debian we cannot add a commented out line in a dsc, use a tag instead
+	  # that can be generated by adding "XS-Obs: needssslcertforbuild" to debian/control
+	  $needsslcert = 1 if /^Obs:\s*needssslcertforbuild\s*$/s;
```

Comment:
shouldn't it be ^XS-Obs: ... ?

Reply:
@adrianschroeter no, that's how it's "coded", X means user-specified tag and "S" means it's only applied to source artefact (ie: the .dsc file), for example:

```
$ grep Obs debian/control 
XS-Obs: needssslcertforbuild
$ grep Obs ../shim_14-1.dsc 
Obs: needssslcertforbuild
```

The OBS backend parses the .dsc files, not debian/control